master/usage/calicoctl.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing and Configuring calicoctl
+---
+
+This document outlines how to install and configure calicoctl which is the
+primary tool for viewing, manipulating, and creating Calico objects on the
+command line.
+
+### Location, location, location
+
+The majority of calicoctl operations only need access to the datastore that
+Calico is using.  This means that the tool can be ran from any host 
+or container that can reach the datastore and has the proper credentials for
+access.
+
+#### Installing calicoctl
+
+The calicoctl tool can be downloaded from the
+[release page of the calicoctl repository]({{site.data.versions[page.version].first.components.calicoctl.url}}),
+set executable, and then it is ready to run.
+
+```
+curl -O -L {{site.data.versions[page.version].first.components.calicoctl.download_url}}
+chmod +x calicoctl
+```
+
+> *Note:* Move calicoctl to a directory in your PATH or add the directory
+  it is in to your PATH to avoid prepending the path to invocations of
+  calicoctl.
+
+#### "Installing" calico/ctl
+
+With each release of calicoctl the docker container `calico/ctl` is released to
+Dockerhub and Quay and can be used for running calicoctl commands.
+Since it is not necessary to "install" a container there is not much to do
+in this section.
+
+If using containerized calicoctl, keep in mind that environment variables
+must be explicitly passed and configuration files must be mounted into the
+container so they are available to the binary.
+
+Due to the containerized nature of this method some of the functionality of
+calicoctl will not work without major configuration efforts (notably the
+`calicoctl node ...` commands will not function correctly).
+
+### Datastore configuration
+
+Follow the setup in the
+[Configuing calicoctl]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup) section.
+That section describes how to do the initial setup of calicoctl, configuring
+the connection information for your Calico datastore.
+
+### After configuration
+
+#### Check the setup
+
+A simple command to check that the installation and configuration is correct
+is `calicoctl get nodes` or
+`docker run --rm -it <configuration> calico/ctl get nodes`.  A correct setup
+will yield a list of the nodes that have registered.  If an empty list is
+returned you are either pointed at the wrong datastore or no nodes have
+registred.  If an error is returned then attempt to correct the issue then
+try again.
+
+### Now the fun starts
```

Comment:
I've definitely worked enough hours on docs that my titles started to take this tone :stuck_out_tongue_closed_eyes: But can we clean up the tone to be a bit more professional? :smile: